Updates readme for more clarification (#39)
RobotsAndPencils/1password-action: README.md

## Outputs

> `Login` items will output both the username and password fields

Reviewer: ```suggestion
`Login` items will output both the username and password fields.
```

`Password` items will output the password field. 

The output variable's name will be the item's name with spaces and `.` replaced with `_`, non-alphanumeric characters removed, and lowercased. For example, a `Password` item named `Google Firebase 2020` would be available as the `google_firebase_2020_password` output variable.

> Another example a `Login` item named `App Store Login` would be available as the `app_store_login_username` and `app_store_login_password` output variables. 

Reviewer: ```suggestion
Another example, a `Login` item named `App Store Login`, would be available as the `app_store_login_username` and `app_store_login_password` output variables. 
```

`Document` items will be saved as files with the same filename as in 1Password. Their filename will be set as an output variable following the same rules as login and password items. For example, a document with the filename `Apple Distribution.p12` will be saved as a file with the same name, and the filename will be available as the output variable `apple_distribution_p12_filename`.

In all cases, you can provide your own output name which will be used in place of the item name from 1Password, and normalized.

> All password outputs are marked with an action mask. 

Reviewer: ```suggestion
All password outputs are marked as secrets so that they're masked in your workflow's logs. 
```
